packages/shared/src/components/profile/devcard/DevCardFooter.tsx

```diff
@@ -21,11 +21,13 @@ export interface DevCardFooterProps {
 export function DevCardFooter({
   type,
   tags,
-  sources,
+  sources = [],
   theme,
   shouldShowLogo = true,
   elementsClickable = true,
 }: DevCardFooterProps): ReactElement {
+  const topSources = sources.slice(0, DevCardConstraints.MaxSourceCount);
```

Comment:
Wouldn't it be better to limit what we query already?
(Not sure top of head where the information comes from now)

Reply:
on the query side limit is already 5

Reply:
So how come we show more then 5? 👀😅

Reply:
👀 what

Reply:
so changes in this PR would do nothing since API will never return more then 5

Reply:
But how come in the issue reported we showed 6??

Reply:
> So how come we show more then 5? 👀😅

I checked on API and it only returns 5, from the image Chris provided in report it actually looks like its just the source image that looks bad

![image](https://github.com/user-attachments/assets/d931441c-1834-4df1-96d0-9d3b742b2600)

Reply:
Oh wow... nice issue 😅 @ilasw u can close this PR and the ticket apparently

Reply:
`DevCardById` query uses `getFavoriteSources` which has `LIMIT 5`.

Reply:
...closing PR!

Reply:
I found it, this is the squad https://app.daily.dev/squads/dotnetangularfullstack

verified locally image looks like that
<img width="386" alt="image" src="https://github.com/user-attachments/assets/977f0d87-8fe0-40ab-ace4-9fd017669018">

Reply:
cc: @nensidosari @ilasw @rebelchris

Reply:
Omg facepalm 🤦‍♂️😢😂